features/update_membership_application.feature

```diff
@@ -1,19 +1,32 @@
-Feature: As an Admin
+Feature: SHF Application status is changed
+
+  As an Admin
   In order to get members into SHF and get their money
-  I need to be able to accept/reject their application
+  I need to be able to review their applications and ultimately accept or reject them
+
   PT: https://www.pivotaltracker.com/story/show/133950603
 
+
+  Note that when counting the total number of status phrases (or words) on the
+  landing page, there will be 1 time where the status will match the
+  *search* list. So if there are 3 applications in a particular state/status,
+  you should expect to see the status *4* times total (3 times for the applications
+  + 1 time in the list of all statuses/states in the search selection list.
```

Comment:
We should hide the search form so that the count of search occurrences matches what the _user_ would see.

Reply:
That _should_ work, except that when  I try that, I still cannot get capybara + selenium + ajax stuff to play well together.

here's what I tried:
1. mark the scenario with @selenium 
2. add these steps at the start (after logging in, going to the right page):
```
    # Hide the search form
    And I click on t("toggle.company_search_form.hide")
    And I wait for all ajax requests to complete
    # verify that it is hidden (otherwise the counts (numbers) will be off below)
    And I should not see t("search")
```

It would fail on the "And I should not see t("search")" step -- the "search" button is still showing up. (it's not really being hidden)

So for now, this passes and is documented well.  
Can we do a different PT chore & PR to address this?

Reply:
I can get it to work by _not_ including the last step above.

I think the step is finding `Sök` via a match of the (show search form (Swedish)) label, ala `Visa Sökfunktionen`.
```
expected not to find text "Sök" in "Skip to content HALLÅ, ADMIN ALLA INKOMNA ANSÖKNINGAR Visa Sökfunktionen ...
```

Reply:
aha! thanks for noting that.  It's working by hiding the search form now.  👍